src/node_contextify.cc

```diff
@@ -1622,7 +1622,7 @@ MaybeLocal<Object> ContextifyFunction::CompileFunctionAndCacheResult(
 // While top-level `await` is not permitted in CommonJS, it returns the same
 // error message as when `await` is used in a sync function, so we don't use it
 // as a disambiguation.
-static std::vector<std::string_view> esm_syntax_error_messages = {
+static const std::vector<std::string_view> esm_syntax_error_messages = {
```

Comment:
Could even do

```suggestion
static const auto esm_syntax_error_messages = std::to_array<std::string_view>({
```

to avoid the heap allocation that will probably still come with this

Reply:
After using std::to_array 
C++20 template extraction gave an error in CI, so I used array, is this healthy? @addaleax

Reply:
I got it, I think we will be able to use it in the future when support comes, unfortunately I got errors in the build step